.gitignore

```diff
@@ -18,3 +18,4 @@ pkg/
 .terraform.lock.hcl
 *.tfvars
 release/wi-kubernetes-manifests.yaml
+release/updated-manifests.yaml
```

Comment:
Let's discard this change :)

Reply:
Then running through the procedure in `spanner.md` will make the working tree non-clean. If that's fine with you it's fine with me :)

Reply:
Oh I see what you mean, but it is the same doc we had in the past fro Memorystore, right? And we didn't have that entry in `.gitignore`? So I think that's ok to discard this here, anyway we will have a Kustomize variation for this, like we have for Memorystore now. But again, that part is out of scope of this current PR.